TINY-9807: No longer delete images on the wrong side of the caret (#8714)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Context toolbars displayed the incorrect status for the `advlist` plugin buttons. #TINY-9680
- On iOS Safari, Korean characters were merging onto the previous line upon typing after inserting a newline by pressing Enter. #TINY-9746
- Initiating the editor with a table at the start would display resize handles even when the editor wasn't focused. #TINY-9748

> - Backspace would in some sitations delete the image in front of it instead of the element behind it as expected. #TINY-9807

Reviewer: ```suggestion
- Backspace would in some situations delete the image after the cursor instead of before it as expected. #TINY-9807
```
